__std__/nix/toolchain/library/cabal-project-index-state.nix

```diff
@@ -1,4 +1,4 @@
 { inputs, cell }:
 
-# TODO revisit why we need this, and why we were parsing it from the actual cabal.project
+# TODO(std) revisit why we need this, and why we were parsing it from the actual cabal.project
 "2022-08-31T00:00:00Z"
```

Comment:
We *definitely* don't want to duplicate it. That will lead to massive confusion. We should define it in one place only, ideally in the `cabal.project`.